packages/graphql-server-lambda/README.md

```diff
@@ -83,3 +83,43 @@ aws cloudformation deploy \
    --stack-name prod \
    --capabilities CAPABILITY_IAM
 ```
+
+## Access or Modify Lambda options
+### Read API Gateway event and Lambda Context
+To read information about the current request (HTTP headers, HTTP method, body, path, ...) or the current Lambda Context (Function Name, Function Version, awsRequestId, time remaning, ...) use the options function. This way they can be passed to your schema resolvers using the context option.
+```js
+var server = require("graphql-server-lambda"),
+    myGraphQLSchema = require("./schema");
+
+exports.graphqlHandler = server.graphqlLambda((event, context) => {
+    const headers = event.headers,
+        functionName = context.functionName;
+
+    return {
+        schema: myGraphQLSchema,
+        context: {
+            headers,
+            functionName,
+            event,
+            context
+        }
+        
+    };
+});
+```
+### Modify the Lambda Response (Enable CORS)
+To enable CORS the response HTTP headers need to be modified. To accomplish this pass in a callback filter to the generated handler of graphqlLambda.
+```js
+var server = require("graphql-server-lambda"),
+    myGraphQLSchema = require("./schema");
+
+exports.graphqlHandler = function(event, context, callback)  {
+    const callbackFilter = function(error, output) {
+        output.headers['Access-Control-Allow-Origin'] = '*';
+        callback(error, output);
+    };
+    const handler = server.graphqlLambda({ schema: myGraphQLSchema });
```

Comment:
I like this example and will use it in my project. It might be a little complex for the README but it does show how to validate domain names instead of just allowing everything. I think I will add one more example that checks the origin of the request. Thanks.